test/ruleLoaderTests.ts

```diff
@@ -18,9 +18,11 @@
 import { assert } from "chai";
 import * as fs from "fs";
 import * as path from "path";
+
 import { rules as allRules, RULES_EXCLUDED_FROM_ALL_CONFIG } from "../src/configs/all";
+import { IOptions } from "../src/language/rule/rule";
 import { camelize } from "../src/utils";
-import { IOptions } from "./../src/language/rule/rule";
+
```

Comment:
was this newline added by the grouped-imports option? what's the logic here? it seems to separate the different levels of imports into groups in some places, but not in others (e.g. in `exclusionFactory.ts`, `../../` imports are in the same group as `../` imports). I haven't been following this rule closely, but that behavior is a little surprising to me and I'd like to revisit it before enabling this rule in `tslint:all`.

Reply:
Yes. There is 3 différents levels: libraries, parents folder and same folder.
```
import something from "module"

import foo from "../../foo"
import bar from "../bar"

import baz from "./baz"
```